### DISCUSSION TITLE
User banned page

### DISCUSSION THREAD

**Discussion by speedyman08** [LBPUnion/ProjectLighthouse#823] [Ideas]
I think that the User Ban Page should specify what the list including "LittleBigPlanetOnline Multiplayer" means. I assume that it means those features are unavailable due to your suspended account. 
![image](https://github.com/LBPUnion/ProjectLighthouse/assets/17880243/3b939073-827b-4868-9add-adfdf3c84583)
should look more like
![image](https://github.com/LBPUnion/ProjectLighthouse/assets/17880243/532285f0-56e8-4cf8-ba09-6b1e99affe88)

**Comment by sudokoko** [comment]
You have actually just encountered a fallback page. This is a known edge case, where you can be marked as banned but have no moderation case be created. I can see where I caused an issue here though, I seem to have forgotten to insert a string like you said. I'll get this fixed.

**Comment by speedyman08** [comment]
You should tell Hyphenator on Discord about it as they are the person that encountered this fallback page
![image](https://github.com/LBPUnion/ProjectLighthouse/assets/17880243/a7ea2dcb-f3b0-40e8-8174-dabcfd81f9a4)

**Comment by sudokoko** [comment]
FIx proposed in #824.
